portfolio/src/main/webapp/index.html

```diff
@@ -45,9 +45,9 @@ <h2>LinkedIn and Github</h2>
             <p>Click here to get a random fact:</p>
             <button onclick="addRandomFact()">Press</button>
             <div id="fact-container"></div>
-            <p>Click here to get a message from a servlet.</p>
-            <button onclick="fetchMessage()">Click</button>
-            <div id="message-container"></div>
+            <h2>Comments</h2>
+            <div id="comments-container"></div>
+            <br>
```

Comment:
Nit: Let's separate this section by newline above and below to make it easier to read the code.

Reply:
Added new line.